engine-rest/engine-rest/src/main/java/org/camunda/bpm/engine/rest/dto/repository/DeploymentWithDefinitionsDto.java

```diff
@@ -31,6 +32,7 @@
   protected Map<String, CaseDefinitionDto> deployedCaseDefinitions;
   protected Map<String, DecisionDefinitionDto> deployedDecisionDefinitions;
   protected Map<String, DecisionRequirementsDefinitionDto> deployedDecisionRequirementsDefinitions;
+  protected Date activationTime;
```

Comment:
At this point, I would suggest to revert the changes applied for `DeploymentWithDefinitionsDto`. For more details, have a look at the general comment for requested changes.